calico-cloud/image-assurance/exclude-vulnerabilities-from-scan-results.mdx

```diff
@@ -0,0 +1,81 @@
+---
+description: Reduce noise in your Image Assurance scan results by excluding vulnerabilities.
+---
+
+import UploadIcon from '/img/icons/upload-icon.svg';
+
+# Exclude vulnerabilities from scan results
+
+This guide shows you how to exclude vulnerabilities from your image scan results.
+You may find this useful reduce the number of incoming vulnerability reports for vulnerabilities that don't require action (for example, false positives).
```

Comment:
I'm not sure about this usage of "reports" here: 

> ... reduce the number of incoming vulnerability **reports** for vulnerabilities that don't require ... 

It sounds like your using a "report" to mean each CVE that is detected and associated with an image. It feels like it is a usage of a specific term for report.

It is also quite wordy with vulnerability mentioned twice in quick succession.

What if we just said this instead: 
> ... reduce the number of reported vulnerabilities that don't require ...